src/lambda/snsToSlack.ts

```diff
@@ -0,0 +1,102 @@
+import * as https from 'https';
+import * as util from 'util';
+
+interface PostDataAttachment {
+  color?: string;
+  text?: string;
+  fields?: {
+    title: string;
+    value: string;
+    short: boolean;
+  }[];
+}
+
+interface SlackHookPostData {
+  username: string;
+  text: string;
+  icon_emoji: string;
+  attachments: PostDataAttachment[];
+}
+
+export const snsToSlackHandler = async (event: any, context: any): Promise<void> => {
+  const { SLACK_WEBHOOK } = process.env;
```

Comment:
Move this outside of the function and mark it as string like this as it is expected to always exists

```
const slackWebhook = process.env.SLACK_WEBHOOK  as string;
```